doc/man1/timew-export.1.adoc

```diff
@@ -5,13 +5,15 @@ timew-export - export tracked time in JSON
 
 == SYNOPSIS
 [verse]
-*timew export* [_<range>_] [_<tag>_**...**]
+*timew export* [_<interval>_**...**] | [[_<range>_] [_<tag>_**...**]]
 
 == DESCRIPTION
-Exports all the tracked time in JSON format.
-Supports filtering.
+Exports all the tracked time in JSON format.  Supply either a list of
+intervals (e.g. `@1 @2`), or optional filters (see **timew-ranges(7)** and/or
+**timew-tags(1)**)
 
 == EXAMPLES
 For example:
```

Comment:
Add a description for each example

Reply:
didn't add this, it didn't already have description for single existing example, and I feel both examples are self-evident, it's subjective though.